docs/source/usage/index.rst

```diff
@@ -24,13 +32,18 @@ ESI will typically be used to fulfill one of two broad scenarios:
 * **Multiple Node Owners Pool Nodes in ESI**
    * In this scenario, multiple node owners pool their hardware together in a single ESI instance. The owners can then use their hardware themselves, or use the mechanisms described in `Lessee Node Assignment`_ to give others access to their nodes.
 
+General Information
+-------------------
+
 .. toctree::
    :maxdepth: 2
 
+   new_user_guide
    lease_management
    network_scenarios
    openshift
    keylime
    cli
 
+.. _New User Guide: new_user_guide.html
 .. _Lessee Node Assignment: common_scenarios.html#lessee-node-assignment
```

Comment:
I was just randomly clicking the links in the doc and found that common_scenarios.html#lessee-node-assignment might be out-dated.

Reply:
I just double-checked - it seems correct to me? I might be missing something

Reply:
I don't see common_scenarios.html. But there is a lease_management.html file generated in build/html/usage.

Reply:
Ah, you're right! Nice catch; fixed!